src/translib/transformer/xlate_from_db.go

```diff
@@ -631,7 +631,9 @@ func yangListInstanceDataFill(dbs [db.MaxDB]*db.DB, ygRoot *ygot.GoStruct, uri s
 				log.Infof("Error returned by %v: %v", xYangSpecMap[xpath].xfmrFunc, err)
```

Comment:
Why are you committing transformer changes as part of this pull request?

Reply:
As mentioned in the description, This PR uses fixes from "transformer-phase2" branch. We'll hold merge of this PR  until "transformer-phase2" is merged into "dell_sonic" branch.